spring-cloud-aws-sqs/src/main/java/io/awspring/cloud/sqs/listener/AbstractContainerOptions.java

```diff
@@ -47,12 +48,16 @@ public abstract class AbstractContainerOptions<O extends ContainerOptions<O, B>,
 
 	private final Duration maxDelayBetweenPolls;
 
+	private final Duration standbyLimitPollingInterval;
```

Comment:
Will we be using this configuration anywhere? If not, we could remove it from here and leave it as a BPH implementation-specific configuration, rather than a Container option.

Wdyt?

Reply:
I removed it for the moment, let's see if later re-introduce it